src/main/resources/db/migration/V20220525_228__HMAN-228-CreateHearingChannelsTable.sql

```diff
@@ -0,0 +1,19 @@
+CREATE SEQUENCE public.hearing_channels_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+CREATE TABLE public.hearing_channels (
+                        hearing_channels_id bigint PRIMARY KEY,
+                        hearing_channel_type varchar(70) not null,
+                        case_hearing_id bigint not null
+);
+
+ALTER SEQUENCE public.hearing_channels_id_seq OWNED BY public.hearing_channels.hearing_channels_id;
+
```

Comment:
can this be added too 
ALTER TABLE ONLY public.hearing_channels ALTER COLUMN id SET DEFAULT nextval('public.hearing_channels_id_seq'::regclass);